src/components/Card/components/CardItem/CardItem.tsx

```diff
@@ -1,25 +1,23 @@
-import { FC, useContext} from "react"
-import ContextStore,{ CardState } from "./actionContext"
+import { FC } from "react";
 import Card from '@mui/material/Card';
 import CardContent from '@mui/material/CardContent';
 import CardMedia from '@mui/material/CardMedia';
 import Typography from '@mui/material/Typography';
 import { CardActionArea } from '@mui/material';
 import Grid from '@mui/material/Grid';
-import { cardStyles } from "./CardStyle";
+import { cardStyles } from "../../cardStyles";
+import { cardsProps } from "../../../../helpers/interface";
```

Comment:
we start names of types and instrfaces from capital letter
and add I before interface and Type in the type to define difference with components